src/typography/typography.css

```diff
@@ -64,6 +68,39 @@ body {
   color: var(--omrs-color-brand-black);
 }
 
+.omrs-type-body-large {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 1.188rem;
+  line-height: 129%;
+  color: var(--omrs-color-brand-black);
+}
+
+.omrs-type-body-regular {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 1.0625rem;
+  line-height: 147.6%;
+  color: var(--omrs-color-brand-black);
+}
+
+.omrs-type-body-serif {
```

Comment:
I think there's a typo in the JIRA issue and that this css class should not exist. When you want serif font, you should do `<div class="omrs-type-body omrs-type-serif"></div>`. Not `<div class="omrs-type-body-serif"></div>`

Reply:
This was not addressed in your most recent commit ^^

Reply:
Removed and recommitted